auth-connector/src/main/java/com/sequenceiq/cloudbreak/auth/altus/GrpcUmsClient.java

```diff
@@ -486,6 +487,16 @@ private boolean makeCheckRightCall(String userCrn, String right, String resource
             client.checkRight(RequestIdUtil.getOrGenerate(requestId), userCrn, right, resource);
             LOGGER.info("User {} has right {} on resource {}!", userCrn, right, resource != null ? resource : "account");
             return true;
+        } catch (StatusRuntimeException statusRuntimeException) {
+            if (Status.Code.DEADLINE_EXCEEDED.equals(statusRuntimeException.getStatus().getCode())) {
+                LOGGER.error("Deadline exceeded for check right {} for user {} on resource {}", right, userCrn, resource != null ? resource : "account",
+                        statusRuntimeException);
+                throw new CloudbreakServiceException("User management service call timed out.");
+            } else {
+                LOGGER.error("Status runtime exception while checking right {} for user {} on resource {}", right, userCrn, resource != null ? resource :
+                        "account", statusRuntimeException);
+                throw new CloudbreakServiceException("Authorization failed due to failed user management service call.");
+            }
```

Comment:
until this point we considered ums failure as denied permission check, do we want to change that from now? Also we should not expose this information about user management failure, this is an internal component.

Reply:
of course logging is useful and needed

Reply:
I discussed it with @biharitomi and it is very misleading to have missing permission errors if we UMS can't respond because it is a false statement. If I would be a user I tried to understand why I don't have permission. I checked also hasRights and we don't handle exceptions there, so exception is thrown out in that case and we don't return false in that case.

Reply:
Yeah we discussed it that may an internal server error is more feasible and less misleading here than a 401 or 403 response.

Reply:
yes it makes sense